instaclustr/resource_kafka_user.go

```diff
@@ -83,7 +58,7 @@ func resourceKafkaUserCreate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("[Error] Cluster %s is not RUNNING.", cluster_id)
 	}
 		
-	// just use linear search for now
+	// just use linear search for now to check if the user going to be created is already in the system
```

Comment:
For the block -
```
for _, str := range usernameList {
		if str == username {
			// user is already set, so we don't change anything
			d.SetId(fmt.Sprintf("%s-%s", cluster_id, username))
			log.Printf("[INFO] Kafka user %d already exists in %s.", username, cluster_id)
		}
	}
```
Could this exit the method early? Otherwise it will continue with the create kafka user request.